File: db/dbshow.py
import mysql.connector


print ("Connecting to MySQL....");

try:
  cnx = mysql.connector.connect(user='takano', password='takanolab',
                              host='localhost',
                              database='dbtest')
except mysql.connector.Error as err:
  print("DB connection error {}".format(err) )

cur = cnx.cursor(dictionary=True)

cur.execute("SELECT id, name, ename FROM student")

for row in cur:
    print(row["id"], row["name"], row["ename"])

cur.close()

cnx.close()

print("Exit.")

File: db/dbshow.php
<?php
print ("Connecting to MySQL....");
try {
  $db = new PDO ('mysql:host=localhost; dbname=dbtest; charset=utf8', 'takano', 'takanolab');
} catch (PDOException $e) {
    exit('DB connection error.'.$e->getMessage());
}

$query = "select * from student";

foreach ($db->query($query) as $row) {
     printf("%s, %s, %s", 
       $row['id'], $row['name'], $row['ename']);
     print("<br>");
 }

$db = null;
print ("Exit.");
?>
